common/static/js/xblock/core.js

```diff
@@ -69,7 +69,7 @@
         } else {
             block = {};
         }
-        block.element = element;
+        block.element = $element;
```

Comment:
@bradenmacdonald The rationale was that all XBlocks inside edx-platform I looked at required the `element` attribute to be a jQuery object.  This was the only way I managed to get it working at all.

Thinking about it, it seems inconsitent to pass a DOM element to the initialisation function and use a jQuery object here, so this line should probably be reverted.  However, this will require a large number of changes in other places, but it's probably what we need to do.